GVFS/GVFS.Service/Mac/org.vfsforgit.service.plist

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Label</key>
+    <string>org.vfsforgit.service</string>
+    <key>ProgramArguments</key>
+    <array>
+        <string>/usr/local/share/dotnet/dotnet</string>
+        <string>/usr/local/vfsforgit/GVFS.Service.dll</string>
+    </array>
+    <key>WorkingDirectory</key>
+    <string>/usr/local/vfsforgit</string>
+    <key>ProcessType</key>
+    <string>Interactive</string>
+    <key>Disabled</key>
```

Comment:
I don't know what's more common on macOS, but I've seen a few example plists like:

```xml
    <key>Name1</key>    <string>Value1</string>
    <key>Name2</key>    <true/>
```
That's a bit easier for me to read over, but definitely not worth blocking the PR for.

Reply:
will take care of this in next PR. Tracking [issue](https://github.com/Microsoft/VFSForGit/issues/979).